src/Bart/Diesel.php

```diff
@@ -44,6 +51,46 @@ public static function create()
 	}
 
 	/**
+	 * Return true when PHPUnit test generator is loaded.
+	 * See comments in __callStatic method for more details.
+	 *
+	 * @return bool
+	 */
+	private static function isTestGeneratorRun()
+	{
+		if (self::$isTestGeneratorRun !== null) {
```

Comment:
I typically flip the logic here, instead checking if the value has *not* been set and if so, then set it. That reduces complexity of the function such that there is only one return statement.